tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -240,22 +255,34 @@ Handle Server Is Stopping
    Sleep  10
    Handle Server Not Running
 
+Control Panel Is Visible
+   [Documentation]  Checks if Control Panel page is open
+   ${control_panel_visible} =  Run Keyword And Return Status  Page Should Contain  Notebook server control panel
+   [Return]  ${control_panel_visible}
+
+Handle Control Panel
```

Comment:
would it be better to call it `Stop Notebook Server From Control Panel`?

Reply:
The other keywords were called `Handle [...]` so I kept it this way for consistency

Reply:
hmm okay. I'd change it, but don't have a strong preference here

Reply:
so, if you think we are going to need a keyword to stop the notebook from control panel for other cases, then I'd change the title. Otherwise I think it's fine

Reply:
That will depend on how the UX evolves. I would expect some way to close down the server from within the JL interface. If that turns out to be a link back to the control panel then I agree with you

Reply:
okay, so let's leave it as it is now. Will change later if needed